tf_fastmri_data/dataset_builder.py

```diff
@@ -18,12 +18,14 @@ def __init__(
             multicoil=False,
             slice_random=False,
             contrast=None,
+            split_slices=False,
             af=4,
             shuffle=False,
             seed=0,
             prebuild=True,
             repeat=True,
             n_samples=None,
+            output_shapes=False,
```

Comment:
actually, let's make this default to None and set it to different default values depending on the value of brain (we should do the same with multicoil tbh but we can do it in a different PR).
```suggestion
            output_shapes=None,
```